src/main/kotlin/gg/essential/universal/UMouse.kt

```diff
@@ -27,6 +60,48 @@ object UMouse {
                 return UResolution.windowHeight - Mouse.getY().toDouble() - 1
                 //#endif
             }
+
+        /** Return the change in the mouse's X position since the last time this method was called. */
+        @JvmStatic
+        val deltaX: Double
+            get() {
+                //#if MC>=11502
+                    //#if FORGE==1
+                    //$$ return UMinecraft.getMinecraft().mouseHelper.getXVelocity()
+                    //#else
+                    //$$ return (UMinecraft.getMinecraft().mouse as MouseAccessor).getCursorDeltaX()
+                    //#endif
+                //#else
+                return Mouse.getDX().toDouble()
+                //#endif
+            }
+
+        /** Return the change in the mouse's Y position since the last time this method was called. */
+        @JvmStatic
+        val deltaY: Double
+            get() {
+                //#if MC>=11502
+                    //#if FORGE==1
+                    //$$ return UMinecraft.getMinecraft().mouseHelper.getXVelocity()
```

Comment:
Minor nitpick, this returns the wrong value

Reply:
oops!